popolo_sources/management/commands/popolo_sources_update.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import print_function, unicode_literals
+
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.six.moves.urllib.parse import urlsplit
+
+from popolo_sources.models import PopoloSource
+from popolo_sources.importer import PopoloSourceImporter
+
+
+REQUIRED_ARG = 'POPOLO-JSON-URL | POPOLO-SOURCE-ID'
+
+
+def is_url(s):
+    try:
+        split_url = urlsplit(s)
+    except ValueError:
+        return False
+    return bool(split_url.scheme)
+
+
+def get_source(source_arg):
+    if re.search(r'^\d+$', source_arg):
+        source_id = int(source_arg, 10)
```

Comment:
Oh, it turns out I was under a misapprehension about how `int` behaved - I thought that if you didn't specify a base then `010` was interpreted as octal, but actually `int('010')` is `10`. I'll take that out :)